pkg/controller/compliancescan/config.go

```diff
@@ -248,47 +247,18 @@ func defaultOpenScapEnvCm(name string, scan *compv1alpha1.ComplianceScan) *corev
 		cm.Data[OpenScapTailoringDirEnvName] = OpenScapTailoringDir
 	}
 
-	if scan.Spec.HTTPSProxy != "" {
-		cm.Data[HTTPSProxyEnvName] = scan.Spec.HTTPSProxy
-	}
-
-	if scan.Spec.NoExternalResources {
-		cm.Data[DisconnectedInstallEnvName] = "true"
-	}
```

Comment:
What happened to the proxy settings and external resources flag?

Reply:
oops. I took the platform function as a base for the common one and forgot to copy over the disconnected install flags.